HW_10/controller.py

```diff
@@ -0,0 +1,15 @@
+import view
+import model
+from telegram.ext import ApplicationBuilder, CommandHandler, ContextTypes
```

Comment:
Пожалуйста, следите за импортами
```suggestion
import view
from telegram.ext import ApplicationBuilder, CommandHandler
```